geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/CacheClientProxyStats.java

```diff
@@ -128,7 +133,10 @@
             "operations"),
 
         f.createLongCounter(CQ_COUNT, "Number of CQs on the client.", "operations"),
-        f.createLongCounter("sentBytes", "Total number of bytes sent to client.", "bytes"),});
+        f.createLongCounter("sentBytes", "Total number of bytes sent to client.", "bytes"),
+        f.createIntCounter(NUMBER_THREADS_PUT_IN_QUEUE,
+            "Number of threads currently adding message in queue", "operations"),
```

Comment:
I would change this description slightly to "Threads currently adding a message to the queue. Consistently high values indicate that the queue is full and adds are being delayed". You might want to change the units to be "threads".